src/model/world/world.h

```diff
@@ -90,6 +90,9 @@ class World {
     /** Kills the Individual with the given ID. Returns whether this operation was successful */
     bool kill(long ID);
 
+    /** Copies the given Individual into the world */
```

Comment:
copy?

Reply:
no, actually pass-by-pointer-reference, you're right.
The previous implementation I had in mind would have copied it over, due to the problem of setting the correct ID

Reply:
added to the docs PR, #51